test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -202,6 +203,12 @@ def test_str(h):
     assert str(h) == expected
 
 
+def test_change_diagram_builder(h):
+    Circuit.default_diagram_builder = BoxDrawingCircuitDiagram
```

Comment:
As above, I think that this new one should be the default, and to change it, customers may need to call circuit.diagram(AsciiCircuitDiagram)